ydb/core/protos/kqp.proto

```diff
@@ -130,6 +130,10 @@ message TKqpPathIdProto {
     optional uint32 TableId = 2;
 }
 
+message TVectorIndexKmeansTreeDescription {
```

Comment:
May I reuse it from flat_scheme_op.proto ?
https://github.com/ydb-platform/ydb/blob/a12ce1c32f4ce8e9461b627f85a814b32f58a32d/ydb/core/protos/flat_scheme_op.proto#L991